### PR TITLE
Release 1.0.0 changes

### PR DESCRIPTION
**Pull request by roman-kupriyanov**
### Added

- Telemetry event recording.
- Added a `[MAP]` folder in the tree view. It is being populated with elements found up the map.
- Added a button `Show logs` in the notification panel which allows the user to open the extension's output.

### Changed

- Multiple error logging and notification events.

### Fixed

- Fixed an issue in the URI building process. Characters that may potentially break the building process are now being escaped.
- Upload dialogs now prevent wildcards as input.
- Fixed an issue with conflict resolution causing the command to fail when resolving reocurring conflicts.
- Fixed an issue with retrieving, editing multiple elements with auto signout option. Now it processes all the selected elements in groups by each location profile.
- Fixed an issue with showing multiple elements for editing. An "editor already opened" error does not appear anymore.
- Incorrect editors closure when discarding changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,27 @@
 
 You can find all notable changes to Explorer for Endevor in this document.
 
+## [1.0.0] &ndash; 2022-02-23
+
+### Added
+
+- Telemetry event recording.
+- Added a `[MAP]` folder in the tree view. It is being populated with elements found up the map.
+- Added a button `Show logs` in the notification panel which allows the user to open the extension's output.
```
Comment (IgorCATech) [inline CHANGELOG.md:11]:
```suggestion
- Added a `Show logs` button to the notification panel, which enables you to open the output of the extension.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,27 @@
 
 You can find all notable changes to Explorer for Endevor in this document.
 
+## [1.0.0] &ndash; 2022-02-23
+
+### Added
+
+- Telemetry event recording.
+- Added a `[MAP]` folder in the tree view. It is being populated with elements found up the map.
```
Comment (IgorCATech) [inline CHANGELOG.md:10]:
```suggestion
- Added a `[MAP]` folder to the tree view. You can use the folder to store the elements that are found up the map.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,27 @@
 
 You can find all notable changes to Explorer for Endevor in this document.
 
+## [1.0.0] &ndash; 2022-02-23
+
+### Added
+
+- Telemetry event recording.
```
Comment (IgorCATech) [inline CHANGELOG.md:9]:
```suggestion
- Added the telemetry event-recording functionality.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,27 @@
...
+- Added a `[MAP]` folder in the tree view. It is being populated with elements found up the map.
+- Added a button `Show logs` in the notification panel which allows the user to open the extension's output.
+
+### Changed
+
+- Multiple error logging and notification events.
+
+### Fixed
+
+- Fixed an issue in the URI building process. Characters that may potentially break the building process are now being escaped.
```
Comment (IgorCATech) [inline CHANGELOG.md:19]:
```suggestion
- Fixed an issue in the URI building process. Characters that may potentially break the building process are escaped.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,27 @@
...
+- Upload dialogs now prevent wildcards as input.
```
Comment (IgorCATech) [inline CHANGELOG.md:20]: Is that a bug fix or a change?

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,27 @@
...
+- Fixed an issue with conflict resolution causing the command to fail when resolving reoccurring conflicts.
```
Comment (IgorCATech) [inline CHANGELOG.md:21]:
```suggestion
- Fixed an issue that caused the conflict resolution command to fail when resolving reoccurring conflicts.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,27 @@
...
+- Fixed an issue with retrieving, editing multiple elements with auto signout option. Now it processes all the selected elements in groups by each location profile.
```
Comment (IgorCATech) [inline CHANGELOG.md:22]:
```suggestion
- Fixed an issue with retrieving and editing multiple elements with the auto signout option. Now auto signout processes all the selected elements in groups by location profile.
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -2,6 +2,27 @@
...
+- Fixed an issue with showing multiple elements for editing. An "editor already opened" error does not appear anymore.
+- Incorrect editors closure when discarding changes.
```
Comment (IgorCATech) [inline CHANGELOG.md:24]:
```suggestion
- Fixed an issue that caused editors to close when discarding changes.
```

File: README.md [hunk 9]
```diff
@@ -49,11 +60,11 @@ Ensure that you meet the following prerequisites before you use Explorer for End
 
 **Host-side prerequisites**:
 
-- Endevor version 18.0.12 or higher with the following PTFs applied:
```
Comment (IgorCATech) [inline README.md:52]: That comma after the number is redundant

File: CHANGELOG.md (hunk 5 above)
Comment (roman-kupriyanov) [reply CHANGELOG.md:20]: I would vote for a bugfix.

@nickImbirev what do you think?

File: README.md (hunk 9 above)
Comment (roman-kupriyanov) [reply README.md:52]: Could you specify exactly where is that comma? I can see it only in this line: `(v18.0.x only) PTFs SO09580, SO09581, SO10013, and SO11268`, but the entire line was completely removed.

File: CHANGELOG.md (hunk 7 above)
Comment (IgorCATech) [reply CHANGELOG.md:22]: @roman-kupriyanov That sounds good to me. Great suggestion

File: CHANGELOG.md (hunk 5 above)
Comment (IgorCATech) [reply CHANGELOG.md:20]: If it's a bug fix, then it should read:

```suggestion
- Fixed an issue that erroneously allowed upload dialogs to use wildcards as input.
```

If not a bug fix, then discard the suggestion

File: README.md (hunk 9 above)
Comment (IgorCATech) [reply README.md:52]: I'm sorry, this should be discarded since you've removed lines 52-56.

File: CHANGELOG.md (hunk 7 above)
Comment (roman-kupriyanov) [reply CHANGELOG.md:22]:
```suggestion
- Fixed an issue with retrieving and editing multiple elements with the auto signout option. Now the retrieve and edit commands correctly process all the selected elements in groups by location profiles.
```

**Review by IgorCATech — APPROVED** [review]
LGTM!

File: README.md [hunk 10]
```diff
@@ -49,11 +60,11 @@ Ensure that you meet the following prerequisites before you use Explorer for End
...
-  - (v18.0.x only) PTFs SO09580, SO09581, SO10013, and SO11268
-  - (v18.1.x only) PTF SO11272
-  - (Optional, Web Services with STC Pooling only) PTFs SO03928 and SO03929
-- Endevor Web Services v2 installed and configured.
+- Endevor version 18.1 with the following PTFs:
+
+  - SO15978
```
Comment (IgorCATech) [inline README.md:65]: Obviously, this can be removed now

Comment (roman-kupriyanov) [inline README.md:65]:
```suggestion
- Endevor version 18.1 with the SO15978 PTF.
```

**Pull request merged**